tools/validators/instance_validator/validate/entity_instance.py

```diff
@@ -362,10 +362,19 @@ def _FieldTranslationIsValid(self, qualified_field_name: str,
         return False
 
       is_valid = True
-      for state in ft.states.keys():
+      raw_values_found = set()
+      for state, value in ft.states.items():
         if state not in valid_states:
           print(f'Field {qualified_field_name} has an invalid state: {state}')
```

Comment:
I know this is from a previous code, can you extend the error message with something like: `expected {valid_states}`

Reply:
Added